### PR TITLE
2167 huawei password breadcrumb

### PR DESCRIPTION
**Pull request by nigel-lowry**
### Context

https://trello.com/c/XCVq2BM4

### CODE REVIEW

**Comment by nigel-lowry** [comment]
[Update breadcrumb for 'Your Huawei router passwords'](https://trello.com/c/XCVq2BM4/2167-update-breadcrumb-for-your-huawei-router-passwords)

File: app/views/huawei_router_passwords/new.html.erb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <%- title = t('page_titles.your_huawei_router_password') %>
 <% content_for :title, title %>
 <%- content_for :before_content do %>
-  <% breadcrumbs([{ "Home" => root_path },
-                  { t('page_titles.signed_in_internet_access') => last_breadcrumb_path_for_huawei },
```
Comment (jits) [inline app/views/huawei_router_passwords/new.html.erb:5]: Can this i18n key (`page_titles.signed_in_internet_access`) be removed now? Of repurposed for the new copy?

Comment (nigel-lowry) [reply app/views/huawei_router_passwords/new.html.erb:5]: Great spot, but that i18n key is still used elsewhere

Comment (jits) [reply app/views/huawei_router_passwords/new.html.erb:5]: Ah good to know! 👍🏽

**Review by jits — APPROVED** [review]
LG!

**Pull request merged**